Source/WebCore/Modules/modern-media-controls/controls/layout-traits.js

```diff
@@ -36,6 +36,11 @@ class LayoutTraits
         return this.mode == LayoutTraits.Mode.Fullscreen;
     }
 
+    isApplePlatform()
```

Comment:
instead of having this explicitly name "Apple", can we have it be specific to the thing it's actually guarding (i.e. `supportsControlBarOverflow`)?

Reply:
I replaced this with `supportsPiP()`. Overflow UI is supported now :)